Stylesheets/PageStyles/wishlist.css

```diff
@@ -0,0 +1,99 @@
+.main {
+    grid-template-columns: auto;   
+}
+
+.background {
+    grid-template-columns: repeat(2, 1fr);
+    filter: blur(3px);
+    position: fixed;
+    z-index: 0;
+    top: 0;
+}
+
+.product-houses {
+    width: 50vw;
+    height: 50vh;
+}
+
+.product-gryffindor {
+    background-color: #800000;  
```

Comment:
you can use color  value in var()

Reply:
Yes I will change them, more files need to be change accordingly in other files too